maintenance_stock/data/demo_maintenance_stock.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="location_inventory" model="stock.location">
+            <field name="name">Inventory adjustment</field>
+            <field name="usage">inventory</field>
+
+        </record>
+        <record id="stock_warehouse_maintenance" model="stock.warehouse">
+            <field name="name">Maintenance Warehouse</field>
+            <field name="partner_id" ref="stock.res_partner_address_41" />
+            <field name="code">Main</field>
+            <field name="reception_steps">one_step</field>
+            <field name="delivery_steps">one_step</field>
```

Comment:
This delivery step seems not valid. I don't see it in the available options for that field
https://github.com/odoo/odoo/blob/1bd9f69f3f3badb7b2eb0cb81b2df9ffea71020f/addons/stock/models/stock_warehouse.py#L59